CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and the project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.
 ### Changed
 
 - Suppress the remaining noisy logs and warnings coming from single-step models ([#53](https://github.com/microsoft/syntheseus/pull/53)) ([@kmaziarz])
+- Removed PredictionList objects, replacing them with Sequence[Prediction] ([#60](https://github.com/microsoft/syntheseus/pull/60)) ([@austint])
```

Comment:
Also, looking at [Common Changelog](https://common-changelog.org/#24-change-group), it has a fourth group of changes called "Removed" (placed between "Added" and "Fixed") for removed functionality. In this case, this PR is almost a refactoring one, but we do remove the possibility of having metadata at the level of an entire list of predictions, so perhaps the entry should go in the "Removed" section.

Reply:
Good suggestion, but this will probably be edited in the future to account for the full merger of reaction model classes, so let's not worry too much about the details here.